### PR TITLE
Use dscp-node types package

### PR DESCRIPTION
**Pull request by jonmattgray**
Use the `@digicatapult/dscp-node` package to get types. Always uses latest version of the package (and latest `dscp-node` image for testing)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,9 +31,9 @@
   },
   "homepage": "https://github.com/digicatapult/dscp-ipfs#readme",
   "dependencies": {
-    "@polkadot/api": "^5.9.1",
+    "@digicatapult/dscp-node": "latest",
     "dotenv": "^8.2.0",
-    "envalid": "^7.0.0",
+    "envalid": "7.0.0",
```
Comment (jonmattgray) [inline package.json:36]: there seems to be some bug with default values for `makeValidator` in newer versions of envalid

File: package.json [hunk 2]
```diff
@@ -31,9 +31,9 @@
   },
   "homepage": "https://github.com/digicatapult/dscp-ipfs#readme",
   "dependencies": {
-    "@polkadot/api": "^5.9.1",
+    "@digicatapult/dscp-node": "latest",
```
Comment (mattdean-digicatapult) [inline package.json:34]: Don't use latest here, lets at least assume we can maintain some sort of semver and use `^3.0.1`

File: package.json (hunk 1 above)
Comment (mattdean-digicatapult) [reply package.json:36]: interesting, is there a bug ticket for envalid you've found?

File: package.json (hunk 2 above)
Comment (jonmattgray) [reply package.json]: changed

File: package.json (hunk 1 above)
Comment (jonmattgray) [reply package.json]: reverted

**Review by mattdean-digicatapult — APPROVED** [review]

**Pull request merged**